diagnostic.md

```diff
@@ -48,11 +54,12 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    router will create params ID. get a product w/ params.<name>_id
+    params: 
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+  You use {{outlet}}.
```

Comment:
The outlet is where Ember will render the template that is specified. How we reference that data is through the Model.